exomiser-cli/src/test/java/org/monarchinitiative/exomiser/cli/CommandLineOptionsParserTest.java

```diff
@@ -161,6 +162,17 @@ void parseSampleAndVcf() {
         assertThat(commandLine.getOptionValue("assembly"), equalTo("hg19"));
     }
 
+    @Test
+    void parseOutputFormat() {
+        CommandLine commandLine = CommandLineOptionsParser.parse(
+                "--sample", resource("exome-analysis.yml"),
+                "--vcf", resource("Pfeiffer.vcf"),
+                "--assembly", "hg19",
+                "--output-format", "TSV_GENE,JSON,HTML,TSV_VARIANT");
+        assertTrue(commandLine.hasOption("output-format"));
+        assertThat(Arrays.asList(commandLine.getOptionValues("output-format")), containsInAnyOrder("TSV_GENE", "JSON", "HTML", "TSV_VARIANT"));
```

Comment:
Use `List.of` instead of `Arrays.asList` - this will create an immutable list. It's not critical here, but a good practice to get into.